test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -125,21 +124,15 @@ $testCases = @{
         HelpInstallationPathHome = "$userHelpRoot\en-US"
     }
 
-    "PackageManagement" = @{
-        HelpFiles            = "Microsoft.PowerShell.PackageManagement.dll-help.xml"
-        HelpInfoFiles        = "PackageManagement_4ae9fd46-338a-459c-8186-07f910774cb8_HelpInfo.xml"
-        CompressedFiles      = "PackageManagement_4ae9fd46-338a-459c-8186-07f910774cb8_en-US_helpcontent$extension"
-        HelpInstallationPath = "$PSHOME\Modules\PackageManagement\en-US"
-        HelpInstallationPathHome = "$userHelpRoot\PackageManagement\en-US"
-    }
-
+<# Need to update PowerShellGet information
```

Comment:
File an issue in this repo to track this and add a link.

Reply:
https://github.com/PowerShell/PowerShell/issues/13332